setup.py

```diff
@@ -79,7 +79,7 @@
             "click>=7,<8",  # for chalice
             "CherryPy>=18,<19",
             "Django>=3,<5",
-            "falcon>=2,<4",
+            "falcon>=3,<4",
```

Comment:
Can you revert this change? This is not necessary
```suggestion
            "falcon>=2,<4",
```

Reply:
I wondered about this! :+1: